src/language-html/parser-html.js

```diff
@@ -39,11 +42,84 @@ function ngHtmlParser(
     getHtmlTagDefinition,
   } = require("angular-html-parser/lib/compiler/src/ml_parser/html_tags");
 
-  const { rootNodes, errors } = parser.parse(input, {
+  const parseResult = parser.parse(input, {
     canSelfClose: recognizeSelfClosing,
     allowHtmComponentClosingTags,
     isTagNameCaseSensitive,
+    getTagContentType,
   });
+  const { rootNodes } = parseResult;
+  let { errors } = parseResult;
+
+  if (options.parser === "vue") {
+    const shouldParseAsHTML = (node) => {
+      if (!node) {
+        return false;
+      }
+      if (node.name === "html") {
+        return true;
+      }
+      if (node.name !== "template") {
+        return false;
+      }
+      const langAttr = node.attrs.find((attr) => attr.name === "lang");
+      const langValue = langAttr && langAttr.value;
+      return langValue == null || getParserName(langValue, options) === "html";
+    };
+    if (rootNodes.some(shouldParseAsHTML)) {
+      let secondParseResult;
+      const doSecondParse = () =>
+        parser.parse(input, {
+          canSelfClose: recognizeSelfClosing,
+          allowHtmComponentClosingTags,
+          isTagNameCaseSensitive,
+        });
+      const getSecondParse = () =>
+        secondParseResult
+          ? secondParseResult
+          : (secondParseResult = doSecondParse());
+      const getSameLocationNode = (node) => {
+        const result = getSecondParse();
+        return result.rootNodes.find((_node) => {
+          if (!_node || !_node.startSourceSpan) {
+            return false;
+          }
+          if (
+            node.startSourceSpan.start.offset ===
+            _node.startSourceSpan.start.offset
+          ) {
+            return true;
+          }
+          return false;
+        });
+      };
```

Comment:
```suggestion
      const getSameLocationNode = (node) =>
        getSecondParse().rootNodes.find(
          ({ startSourceSpan }) =>
            startSourceSpan &&
            startSourceSpan.start.offset === node.startSourceSpan.start.offset
        );
```

`node` can't be false, right?